test/unit/api.test.js

```diff
@@ -66,13 +70,26 @@ describe('Public API', function () {
         nqlLang.parse.calledOnce.should.be.true();
     });
 
-    it('Supports options (aliases)', function () {
-        const query = nql('tags:[photo]', {aliases: testAliases});
+    it('Supports options (expansions)', function () {
+        const query = nql('tags:[photo]', {expansions: expansions});
 
         query.toJSON().should.eql({'tags.slug': {$in: ['photo']}});
         query.toString().should.eql('tags:[photo]');
 
         query.queryJSON({tags: [{slug: 'video'}, {slug: 'audio'}]}).should.be.false();
         query.queryJSON({id: 3, tags: [{slug: 'video'}, {slug: 'photo'}, {slug: 'audio'}]}).should.be.true();
     });
+
+    it('Supports options (expansions extended)', function () {
```

Comment:
Added this test to have at least one sample of how nql api should be used with not just alias substitution but with filter expansion as well